mobile/src/translations/en.json

```diff
@@ -78,6 +83,24 @@
         "value": "{{value}} BURST"
       }
     },
+    "dates": {
+      "months": {
+        "short": {
+          "0": "Jan",
+          "1": "Feb",
+          "2": "Mar",
+          "3": "Apr",
+          "4": "May",
+          "5": "Jun",
+          "6": "Jul",
+          "7": "Aug",
+          "8": "Sep",
+          "9": "Oct",
+          "10": "Nov",
+          "11": "Dec"
```

Comment:
this is just asking for trouble IMO - we should either start at 1 or use the month names for brevity.

Reply:
Month names map is a nice idea actually